input/fsh/DEF_CodeSystems.fsh

```diff
@@ -13,7 +13,6 @@ Terms beginning with 'USCRS-' (for United States Content Request Service) repres
 * #1286801002 "Real time tumor tracking external beam radiotherapy (procedure)" //USCRS-34878
 * #1286802009 "External beam radiotherapy using internal target volume approach (procedure)" // #USCRS-34880
 * #721081000124101  "Radiotherapy with abdominal compression (regime/therapy)" "Radiotherapy with abdominal compression (regime/therapy)" //USCRS-34882
-* #721041000124107 "Radiotherapy with respiratory motion management (regime/therapy)" "Radiotherapy with respiratory motion management (regime/therapy )" // USCRS-34885.  Parent code, not directly used by IG.
```

Comment:
Removed. We planned to use it as a parent code, but eventually did not. The motion management codes were linked differently in SNOMED (under motion management with free breathing). Also a SNOMED code was not assigned.